doc/content/reference/telemetry/identity_server.md

```diff
@@ -0,0 +1,89 @@
+---
+title: "Identity Server Telemetry"
+description: "Description of everything that the telemetry procedure collects regarding the IdentityServer"
+aliases: "/telemetry/identity-server"
+---
+
+[The Things Stack]({{<ref "/the-things-stack">}})'s Identity Server collects telemetry data, which is used to help us understand how users interact with the product. For example, the current data helps to have a better insight on the user's usage of organizations. Telemetry collection is enabled by default and does not collect personally identifiable information. Telemetry collection is enabled by default and does not collect personally identifiable information. The following section section explains what information is collected and how it is anonymised.
+
+## Identification
+
+When the Identity Server starts the background task responsible for collecting and sending telemetry data, it generates the UID which is attached to every telemetry message that is sent, its value is the SHA512 hash of `console.ui[IS,GS,NS,AS,JS].BaseURL` values.
+
+{{< note >}}
+For cases where using these values is not acceptable to the user, there is the option to set your own list of elements used to create the SHA512 hash. Set `telemetry.uid-elements` to a list of string values of your choice.
```

Comment:
```suggestion
For cases where using these values is not acceptable to the user, there is an option to set your own list of elements used to create the SHA512 hash. Set `telemetry.uid-elements` to a list of string values of your choice.
```

Reply:
How do you set `telemetry.uid-elements`? Where?

Reply:
Updated the note to reference to the Stack's configuration and left an example in there as well.